source/MaterialXRuntime/RtConnectableApi.cpp

```diff
@@ -16,12 +16,6 @@ namespace
 
 bool RtConnectableApi::acceptConnection(const RtInput& input, const RtOutput& output) const
 {
-    // TODO: Tokens are not currently considered to be connectable
-    if (input.isToken())
-    {
-        return false;
-    }
```

Comment:
Remove this as this check is already done in PvtInput::isConnectable() that call this method.

Reply:
Thanks for the cleanup. BTW, for 1.39 the the desire is to have "interface tokens" connected to tokens on nodes inside graphs so we'll need the connection capability.

Reply:
Yep, I figured connectability was being introduced later since tokens were implemented as inputs rather then a separate port class with plain values and no connections.